### ISSUE TITLE
Resume training, how to change learning scheduler?

### ISSUE THREAD

**Issue by jzhanghzau** [Lightning-AI/pytorch-lightning#19865]
### Bug description

I used the cosine learning scheduler in my first round training and after reaching the set max steps, I stopped the training. And now I need to restart the training and replace the learning rate scheduler with a new one because if I don't, the learning rate should always be 0. My question is how to restarting the training with a new learning rate scheduler? 

I just replace a new learning rate scheduler in` def configure_optimizers()` and then set last checkpoint in` trainer.fit()`?

Thanks so much !

JJ

### What version are you seeing the problem on?

master

### Error messages and logs

```
# Error messages and logs here please
```

### Environment

<details>
  <summary>Current environment</summary>

```
#- Lightning Component (e.g. Trainer, LightningModule, LightningApp, LightningWork, LightningFlow):
#- PyTorch Lightning Version (e.g., 1.5.0):
#- Lightning App Version (e.g., 0.5.2):
#- PyTorch Version (e.g., 2.0):
#- Python version (e.g., 3.9):
#- OS (e.g., Linux):
#- CUDA/cuDNN version:
#- GPU models and configuration:
#- How you installed Lightning(`conda`, `pip`, source):
#- Running environment of LightningApp (e.g. local, cloud):
```

</details>

**Comment by jzhanghzau** [comment]
I only want to load weights, everything else will be initialized, how can I do this?

By the way, I want combine this with lightingCLI?

Thanks in advance!

Bests,

JJ
